spec/system/ticket/update/full_quote_header_spec.rb

```diff
@@ -162,7 +196,7 @@ def citation
     end
 
     def name
-      expected.created_by.fullname
+      expected.origin_by.fullname || expected.created_by.fullname
```

Comment:
Are you sure this won't crash of `origin_by` is `nil`? :)

Reply:
Yep, added a null check for that.